src/OrchardCore/OrchardCore.Search.AzureAI.Core/Services/AzureAISearchIndexingService.cs

```diff
@@ -81,6 +81,7 @@ public async Task ProcessContentItemsAsync(params string[] indexNames)
         var tasks = new List<IndexingTask>();
 
         var allContentTypes = indexSettings.SelectMany(x => x.IndexedContentTypes ?? []).Distinct().ToList();
+        var readOnlySession = _store.CreateSession(withTracking: false);
```

Comment:
Maybe session for simplicity

Reply:
no. I purposely called it `readOnlySession` to indicate that it is a session for read only.